contrib/mac/frameworkapp/Makefile

```
@@ -17,15 +17,34 @@ XCEXPORT_METHOD ?= development
 # Identity for the product archive (the .pkg installer).
 DARWIN_CODESIGN_PRODUCT_ARCHIVE_KEYCHAIN_IDENTITY ?= -
 else
-DARWIN_CODESIGN_KEYCHAIN_IDENTITY ?= Developer ID Application
-APPLE_DEVELOPMENT_TEAM ?= unidentified
+DARWIN_CODESIGN_KEYCHAIN_IDENTITY ?= AFB379C0B4CBD9DB9A762797FC2AB5460A2B0DBE
+APPLE_DEVELOPMENT_TEAM ?= A427R7F42H
```

```suggestion
DARWIN_CODESIGN_KEYCHAIN_IDENTITY ?= Developer ID Application
APPLE_DEVELOPMENT_TEAM ?= unidentified
```
Or, what are these other numbers?